sdk/communication/Azure.Communication.JobRouter/tests/Samples/ClassificationPolicyCrudOps.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Communication.JobRouter.Tests.Infrastructure;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Communication.JobRouter.Tests.Samples
+{
+    public class ClassificationPolicyCrudOps : SamplesBase<RouterTestEnvironment>
+    {
+        [Test]
+        public void ClassificationPolicyCrud()
+        {
+#if !SNIPPET
+            // create a client
+            var routerAdministrationClient = new RouterAdministrationClient(Environment.GetEnvironmentVariable("AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING"));
+#endif
+
+            #region Snippet:Azure_Communication_JobRouter_Tests_Samples_Crud_CreateClassificationPolicy
```

Comment:
This snippet won't compile due to the above `#if` block.   Since that conditional is outside the scope of the snippet region, I don't think we need the conditional there.

Reply:
This has been updated